test/levelup-test.js

```diff
@@ -17,11 +17,11 @@ describe('levelup', function() {
 
 		zstreams
 			.fromArray(testArray)
-			.pipe(db.createWriteStream())
+			.pipe(db.createWriteStream(), { objectMode: true })
```

Comment:
This isn't for auto convertion. `.pipe` already takes [pipeOpts](https://github.com/joyent/node/blob/master/lib/_stream_readable.js#L467)